distill.py

```diff
@@ -0,0 +1,218 @@
+"""
+Distillation Script for Wake Vision and Visual Wake Words Datasets
+"""
+import numpy as np
+import os
+
+os.environ["KERAS_BACKEND"] = "jax"
+
+# Note that keras should only be imported after the backend
+# has been configured. The backend cannot be changed once the
+# package is imported.
+import keras
+
+import tensorflow as tf
+import tensorflow_datasets as tfds
+import numpy as np
+
+from experiment_config import default_cfg, get_cfg
+from wake_vision_loader import get_wake_vision, get_miaps
+from vww_loader import get_vww
+
+import wandb
+from wandb.keras import WandbMetricsLogger
+
+
+def distill(teacher_config, student_cfg=default_cfg, extra_evals=["distance_eval", "miap_eval"]):
```

Comment:
Is there any reason that lighting is not included as an extra eval?

Reply:
No that was just how it was set up originally. 

I'm actually going to remove validation evals for now since it slows things down and they aren't f1. 

We can add them back to distill later but for now we mostly care about the test set versions for finer grain eval.